crates/common_utils/src/lib.rs

```diff
@@ -43,6 +43,28 @@ pub mod date_time {
         let result = block().await;
         (result, start.elapsed().as_seconds_f64() * 1000f64)
     }
+
+    /// Prefix the date field with zero if it has single digit Eg: 1 -> 01
+    fn prefix_zero(input: u8) -> String {
+        if input < 10 {
+            return "0".to_owned() + input.to_string().as_str();
+        }
+        input.to_string()
+    }
```

Comment:
```suggestion
    /// Zero pads the provided input to 2 digits
    fn two_digit_zero_pad(input: u8) -> String {
        format!("{input:02}")
    }
```

Regardless, use the `time` crate's utilities as Narayan suggested.